extensions/sql-database-projects/src/test/netCoreTool.test.ts

```diff
@@ -41,8 +44,33 @@ describe('NetCoreTool: Net core install popup tests', function (): void {
 			should(result).true('dotnet is either not present or in /usr/local/share by default');
 		}
 	});
-});
 
-export async function sleep(ms: number): Promise<{}> {
-	return new Promise(resolve => setTimeout(resolve, ms));
-}
+	it('Should run dotnet command', async function (): Promise<void> {
+		this.timeout(10000 * 10); // higher timeout for this test
+
+		const netcoreTool = new NetCoreTool();
+		const projfile = path.join(__dirname, 'baselines', 'simpleProject.sqlproj');
+		const dacpacfile = path.join(__dirname, 'baselines', 'bin', 'debug', 'simpleProject.dacpac');
```

Comment:
If you use `projFilePath = await testUtils.createTestSqlProj(baselines.openProjectFileBaseline);` and put the dacpac in the same folder, then it'll keep all the test artifacts together, and give them a slight TTL (because they're put in the temp folder) instead of cleaning up immediately (useful if you want to look at them when tests fail).  I think using the user temp folder for this is common in ADS.